common/src/main/java/org/opensearch/ml/common/connector/template/APISchema.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.connector.template;
+
+import lombok.Data;
+import lombok.Getter;
+import org.apache.commons.text.StringSubstitutor;
+import org.opensearch.common.io.stream.NamedWriteable;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
+import static org.opensearch.ml.common.utils.StringUtils.gson;
+
+@Data
+public class APISchema implements ToXContentObject, NamedWriteable {
+    public static final String PARSE_FIELD_NAME = "API_Schema";
```

Comment:
Let's follow the same naming pattern, change to lower case and connect with "_", 
```suggestion
    public static final String PARSE_FIELD_NAME = "api_schema";
```

Reply:
Yes will do in the refresh.